packages/browser/src/index.test.ts

```diff
@@ -3,109 +3,8 @@ import LogtoClient from '.';
-const refreshToken = 'new_refresh_token_value';
-const idToken = 'id_token_value';
-const currentUnixTimeStamp = Date.now() / 1000;
-const storageKey = `logto:${appId}`;
-
-const fetchOidcConfig = jest.fn(async () => {
-  await new Promise((resolve) => {
-    setTimeout(resolve, 0);
-  });
-
-  return {
-    authorizationEndpoint,
-    tokenEndpoint,
-    endSessionEndpoint,
-    revocationEndpoint,
-    jwksUri,
-    issuer,
-  };
-});
-
-jest.mock('@logto/js', () => ({
-  ...jest.requireActual('@logto/js'),
-  fetchOidcConfig: async () => fetchOidcConfig(),
-  decodeIdToken: jest.fn(() => ({
-    iss: 'issuer_value',
-    sub: 'subject_value',
-    aud: 'audience_value',
-    exp: currentUnixTimeStamp + 3600,
-    iat: currentUnixTimeStamp,
-    at_hash: 'at_hash_value',
-  })),
-  generateCodeChallenge: jest.fn(async () => mockCodeChallenge),
-  generateCodeVerifier: jest.fn(() => mockedCodeVerifier),
-  generateState: jest.fn(() => mockedState),
-  verifyIdToken: jest.fn(),
-  fetchTokenByAuthorizationCode: jest.fn(async () => ({
-    accessToken,
-    refreshToken,
-    idToken,
-    scope: 'read register manage',
-    expiresIn: 3600,
-  })),
-}));
-
-jest.mock('./utils/generators', () => ({
-  ...jest.requireActual('./utils/generators'),
-  generateCodeChallenge: jest.fn(async () => mockCodeChallenge),
-  generateCodeVerifier: jest.fn(() => mockedCodeVerifier),
-  generateState: jest.fn(() => mockedState),
-}));
-
-const createRemoteJWKSet = jest.fn(async () => '');
-
-jest.mock('jose', () => ({
-  ...jest.requireActual('jose'),
-  createRemoteJWKSet: async () => createRemoteJWKSet(),
-}));
-
-describe('LogtoClient', () => {
-  afterEach(() => {
-    localStorage.clear();
-    sessionStorage.clear();
-  });
-
-  describe('constructor', () => {
-    it('constructor should not throw', () => {
-      expect(() => new LogtoClient({ endpoint, appId })).not.toThrow();
-    });
-  });
-
-  describe('storage', () => {
-    it('should set tokens to sessionStorage and localStorage properly', async () => {
-      const logtoClient = new LogtoClient({ endpoint, appId });
-      await logtoClient.signIn(redirectUri);
-
-      const code = `code_value`;
-      const callbackUri = `${redirectUri}?code=${code}&state=${mockedState}&codeVerifier=${mockedCodeVerifier}`;
-
-      expect(sessionStorage.getItem(storageKey)).not.toBeNull();
-      await logtoClient.handleSignInCallback(callbackUri);
-      expect(localStorage.getItem(`${storageKey}:refreshToken`)).toEqual(refreshToken);
-      expect(localStorage.getItem(`${storageKey}:idToken`)).toEqual(idToken);
-    });
-  });
-
-  describe('handleRedirect', () => {
-    it('should cause location change after calling signIn', async () => {
-      expect(window.location.toString()).not.toContain(endpoint);
-      const logtoClient = new LogtoClient({ endpoint, appId });
-      await logtoClient.signIn(redirectUri);
-      expect(window.location.toString()).toContain(endpoint);
-    });
```

Comment:
this part is not related to storage, and should keep it?
(removed it since it broke UT for now?)

Reply:
You are right, this part is not related to storage, but still SHOULD be removed, since it requires some mocks in `@logto/js`, and another purpose of this PR is to remove `@logto/js`.